js/index.js

```diff
@@ -40,3 +40,137 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+
+// Code Starts Here ------------------------------------>
+// Images Section
+
+document.getElementById('logo-img').src = 'img/logo.png';
+document.getElementById('cta-img').src = 'img/header-img.png';
+document.getElementById("middle-img").src = 'img/mid-page-accent.jpg';
+
+
+//Button Section
+
+let h1Tag = document.querySelector('h1')
+h1Tag.innerHTML =  siteContent.cta.h1;
+
+
+let btnInfo = document.querySelector('button');
+btnInfo.innerHTML = siteContent.cta.button;
+
+
+//Main Content Section
+
+let topContH4 = document.getElementsByClassName("text-content")
+
+
+let h4 = Array.from(topContH4);
+let contPArray = Array.from(topContH4);
+
+for(let i =0; i < h4.length; i++){
+  h4[i].querySelector('h4').id = `h4-${i + 1}`;
+  contPArray[i].querySelector('p').id = `p-${i + 1}`;
+}
+
+document.getElementById('h4-1').innerHTML = siteContent["main-content"]["features-h4"];
+document.getElementById('h4-2').innerHTML = siteContent["main-content"]["about-h4"];
+document.getElementById('h4-3').innerHTML = siteContent["main-content"]["services-h4"];
+document.getElementById('h4-4').innerHTML = siteContent["main-content"]["product-h4"];
+document.getElementById('h4-5').innerHTML = siteContent["main-content"]["vision-h4"];
+
+document.getElementById('p-1').innerHTML = siteContent["main-content"]["features-content"];
+document.getElementById('p-2').innerHTML = siteContent["main-content"]["about-content"];
+document.getElementById('p-3').innerHTML = siteContent["main-content"]["services-content"];
+document.getElementById('p-4').innerHTML = siteContent["main-content"]["product-content"];
+document.getElementById('p-5').innerHTML = siteContent["main-content"]["vision-content"];
+
```

Comment:
Look at the for loop you've got above this, think about how to reduce the amount of same things your typing, heres one more dynamic solution to this problem
```
for(let i = 0; i< mainContent.length; i++){
  let section = '';
  switch(i){
    case 0: section ='features'
    case 1: section = 'about'
    case 2: section = 'services'
    case 3: section = 'product'
    case 4: section = 'vision'
  }
  mainContent[i].getElementById(`h4-${i+1}`).textContent = siteContent["main-content"][`${section}-h4`]
  mainContent[i].getElementById(`p-${i+1}`).textContent = siteContent["main-content"][`${section}-content`]
}
```
with that switch statement we can change that string value based on the loop number with a switch statment instead of typing it in every time.